extensions/adobe/experience/decisioning/proposition-attribution.schema.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
@jbeckert , Once this gets realigned to provide a core mixin for experienceevent, this mixin should probably be moved to - https://github.com/adobe/xdm/tree/master/components/mixins/experience-event - and renamed to match the other experienceevent mixins for consistency.